src/Apps/Order/Routes/Review/index.tsx

```diff
@@ -51,14 +51,15 @@ export class ReviewRoute extends Component<ReviewProps> {
         ? Schema.ActionType.SubmittedOrder
         : Schema.ActionType.SubmittedOffer,
     order_id: props.order.id,
+    product_id: props.order.lineItems.edges[0].node.artwork.id
```

Comment:
I have the same issue here... the Mongo ID doesn't appear to be available in this artwork representation either.